bin/clean-jenkins-log

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+cat the-build.log \
```

Comment:
Articles in things just tend to be confusing (why is this _the_ build log, as opposed to just _a_ build log?)

Reply:
No problem, will change.

Reply:
Changed with commit 6b2b3ee83b2f572db54dec888f316d963d84b06b